src/components/bottomSheetContentWrapper/BottomSheetContentWrapper.ios.tsx

```diff
@@ -16,7 +19,11 @@ const BottomSheetContentWrapperComponent = forwardRef<
       onGestureEvent={onGestureEvent}
       onHandlerStateChange={onHandlerStateChange}
     >
-      <Animated.View pointerEvents="box-none">{children}</Animated.View>
+      {isScreenReaderEnabled ? (
+        children
```

Comment:
I'm assuming there was a reason to wrap the content in `Animated.View`. If it does not need to be there, then if should be removed altogether.

Is the screen reader having issues with `Animated.View` or with `pointerEvents="box-none"`? (I'd guess it's the latter).

Reply:
I think the problem is with `Animated.View`. The android version of the file does not have it and screen readers work perfect with it https://github.com/gorhom/react-native-bottom-sheet/blob/v2/src/components/bottomSheetContentWrapper/BottomSheetContentWrapper.android.tsx

Also, i think it's fine to not render the animated view if screen reader is enabled since you cannot scroll with it anyways